packages/babylon/src/plugins/jsx/index.js

```diff
@@ -69,8 +69,26 @@ function getQualifiedJSXName(
   throw new Error("Node had unexpected type: " + object.type);
 }
 
+function assert(x: boolean): void {
+  if (!x) {
+    throw new Error("Assert fail");
+  }
+}
+
 export default (superClass: Class<Parser>): Class<Parser> =>
   class extends superClass {
+    jsxTagStartToRelational() {
+      assert(this.match(tt.jsxTagStart));
```

Comment:
Not important for this pr, but we might remove these `assert` calls in the production build.

Reply:
I've already planned to create something like that on one of my project. Should I work on this?